src/LuaTransformer.ts

```diff
@@ -141,7 +141,7 @@ export class LuaTransformer {
     public transformStatement(node: ts.Statement): StatementVisitResult {
         // Ignore declarations
         if (node.modifiers && node.modifiers.some(modifier => modifier.kind === ts.SyntaxKind.DeclareKeyword)) {
-            return undefined;
+            return this.transformDeclareStatement(node);
```

Comment:
These types can be declared in other file that isn't going to be transformed, so it should be handled when it's used